src/components/dialogs/DialogGatewayValue.vue

```diff
@@ -114,6 +114,52 @@
                   v-model="editedValue.verifyChanges"
                 ></v-switch>
               </v-flex>
+
+              <v-flex xs6>
+                <v-switch
+                  label="Parse send"
+                  hint="Create a function that parse the value sent via MQTT"
+                  persistent-hint
+                  v-model="editedValue.parseSend"
+                ></v-switch>
+              </v-flex>
+
+              <v-container v-if="editedValue.parseSend">
+                <br />
```

Comment:
looks like you're using `<br />` to introduce some cosmetic spacing, can that be done with css, would that be better?

Reply:
I missed that. Removed